src/ziggurat/timestamp_transformer.clj

```diff
@@ -19,9 +20,14 @@
 
 (deftype TimestampTransformer [^{:volatile-mutable true} processor-context metric-namespace oldest-processed-message-in-s additional-tags] Transformer
          (^void init [_ ^ProcessorContext context]
-           (do (set! processor-context context)
-               nil))
+           (set! processor-context context))
          (transform [_ record-key record-value]
+           (log/info "------------ record metadata -----------")
```

Comment:
Can we publish record metadata in one log statement instead of 7 different statements?

Reply:
done